### PR TITLE
Close #21569: Remove 'Window' from Window Open function names 

### LINKED ISSUE #21569: Remove unneeded function name Window
Now that windows are all under the OpenRCT2::Ui::Windows namespace there is no need to keep Window in the function name.

https://github.com/OpenRCT2/OpenRCT2/blob/7a60e7f2e09e002c13765380636e108cd2249a08/src/openrct2-ui/windows/Window.h#L40

So all of these Open functions should have Window removed from them.
I.e. `WindowAboutOpen` becomes `AboutOpen`

### PR DESCRIPTION
**Pull request by CoryfY**
Closes #21569.

### CODE REVIEW

File: src/openrct2-ui/WindowManager.cpp [hunk 1]
```diff
@@ -251,41 +251,41 @@ class WindowManager final : public IWindowManager
                 return w;
             }
             case WindowClass::ManageTrackDesign:
-                return WindowTrackManageOpen(
+                return TrackManageOpen(
                     static_cast<TrackDesignFileRef*>(intent->GetPointerExtra(INTENT_EXTRA_TRACK_DESIGN)));
```
Comment (tupaschoal) [inline src/openrct2-ui/WindowManager.cpp:255]:
```suggestion
                return TrackManageOpen(static_cast<TrackDesignFileRef*>(intent->GetPointerExtra(INTENT_EXTRA_TRACK_DESIGN)));
```

File: src/openrct2-ui/WindowManager.cpp [hunk 2]
```diff
@@ -251,41 +251,41 @@ class WindowManager final : public IWindowManager
...
             {
                 const auto rideId = RideId::FromUnderlying(intent->GetSIntExtra(INTENT_EXTRA_RIDE_ID));
                 auto ride = GetRide(rideId);
-                return ride == nullptr ? nullptr : WindowRideMainOpen(*ride);
+                return ride == nullptr ? nullptr : RideMainOpen(*ride);
             }
             case WindowClass::TrackDesignPlace:
-                return WindowTrackPlaceOpen(
+                return TrackPlaceOpen(
                     static_cast<TrackDesignFileRef*>(intent->GetPointerExtra(INTENT_EXTRA_TRACK_DESIGN)));
```
Comment (tupaschoal) [inline src/openrct2-ui/WindowManager.cpp:279]:
```suggestion
                return TrackPlaceOpen(static_cast<TrackDesignFileRef*>(intent->GetPointerExtra(INTENT_EXTRA_TRACK_DESIGN)));
```

**Review by tupaschoal — COMMENTED** [review]
If you take a close look, the CI failed on the [formatting job](https://github.com/OpenRCT2/OpenRCT2/actions/runs/8303987501/job/22729000955?pr=21613). Technically the two suggestions I made fix it.

**Comment by CoryfY** [comment]
Thank you very much. I was very confused why the check failed as it did. 
Suggestions have been applied.

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**